test/MongoOdmTest.php

```diff
@@ -226,6 +235,11 @@ public function basicQueryProvider()
                     array('name' => 'My First Sprocket')
                 )
             ),
+            'like special char field' => array(
+                'like(name,*special(*),limit(1)', array(
```

Comment:
so this is the wrong input.. if we accept urlencoding as value, this should be `like(name,%2Aspecial%28%2A),limit(1)` and we decode at the correct location(s)..